conda.recipe/meta.yaml

```diff
@@ -1,10 +1,13 @@
+{% set data = load_setup_py_data() %}
+
 package:
   name: bokeh
-  version: 'nightly'
+  version: {{ data['version'] }}
+
+source:
+  path: ..
```

Comment:
@damianavila ``path`` is not going away.  The difference is:

``path``: takes the current state of files/folders as they are at the moment of build.  Copies them to a temporary location.
``git_url``: clones the repo on disk.  This is the state at the most recent commit (when no git_tag is specified, at least).  This will not include changes on disk that have not yet been committed.

What will be going away very soon is support for post-build versioning - creating ``__conda_version__.txt`` files.